### ISSUE TITLE
Review FB00000958, FB00000959, FB00000960, FB00000961, FB00000962, and FB00000963

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/facebase-curation#87]
Selleri lab has reported these datasets as ready for Hub review.

**Comment by robes** [comment]
My impressions

FB00000958: good
FB00000959: good
FB00000960: good
FB00000961: good
FB00000962: good
FB00000963: good

They all looked good to me.

**Comment by BridgetDS** [comment]
I think they are fine to release as-is, but at some point it would be helpful (i.e. make them easier to find via the faceted browser) to have some way to tag these with the human/chimp genes that the enhancers are near (FOXI2, COL13A1, NARS2) -- right now they are (not terribly helpfully) listed only under mouse gene panTro4. It also feels odd to have these labeled solely as mouse data, even though that is technically true.

**Comment by robes** [comment]
Right. I agree with that. I want to begin a conversation with them and two other groups doing enhancer assay work and refactor this.

**Comment by bugacov** [comment]
FB958: Looks Good. Might be good to add Protocol to Experiment and local_indentifier to Experiment and/or biosample. If look at the file it has 3XhEt1 and the enhancer Lab Internal field says 3XhET1_pHsp68-lacZ-tdTomato. So maybe the same ID can be use in Experiment.
FB959: Same comment as 958.
FB960: Same comment as 958.
FB961: Same comment as 958.
FB962: Same comment as 958.

I think we can mark them as Curated and ask them if they can add the identifiers (which they already have in the Enhancer assay

**Comment by bugacov** [comment]
@robes  and @BridgetDS ... Not sure what should be done about this issue. 

1) Are we going to try to address the issue that Bridget about tagging w/ other genes? 

2) @robes ... it seems to me that in these datasets we can add a Chromosome entry w/ the start-end position?

**Comment by BridgetDS** [comment]
@bugacov @robes I would say we should ask someone who is knowledgeable about enhancers for suggestions on this (probably Axel Visel, since he is one of our scientific advisors)

**Comment by BridgetDS** [comment]
@bugacov @robes I would say we should ask someone who is knowledgeable about enhancers for suggestions on this (probably Axel Visel, since he is one of our scientific advisors)
